components/underlay/index.css

```diff
@@ -10,50 +10,46 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import '@spectrum-css/commons/overlay.css';
+@import "@spectrum-css/commons/overlay.css";
 
 .spectrum-Underlay {
-  --spectrum-underlay-background-entry-animation-delay: var(--spectrum-animation-duration-0);
-  --spectrum-underlay-background-exit-animation-ease: var(--spectrum-animation-ease-in);
-  --spectrum-underlay-background-entry-animation-ease: var(--spectrum-animation-ease-out);
-  --spectrum-underlay-background-exit-animation-duration: var(--spectrum-animation-duration-100);
-  --spectrum-underlay-background-entry-animation-duration: var(--spectrum-animation-duration-600);
-  --spectrum-underlay-background-exit-animation-duration: var(--spectrum-animation-duration-300);
-  --spectrum-underlay-background-exit-animation-delay: var(--spectrum-animation-duration-200);
-
-    /* TODO update to --spectrum-overlay-color token once an RGB stripped value is available */
-  --spectrum-underlay-background-color: rgba(var(--spectrum-black-rgb), var(--spectrum-overlay-opacity));;
+	--spectrum-underlay-background-entry-animation-delay: var(--spectrum-animation-duration-0);
+	--spectrum-underlay-background-exit-animation-ease: var(--spectrum-animation-ease-in);
+	--spectrum-underlay-background-entry-animation-ease: var(--spectrum-animation-ease-out);
+	--spectrum-underlay-background-exit-animation-duration: var(--spectrum-animation-duration-100);
+	--spectrum-underlay-background-entry-animation-duration: var(--spectrum-animation-duration-600);
+	--spectrum-underlay-background-exit-animation-duration: var(--spectrum-animation-duration-300);
+	--spectrum-underlay-background-exit-animation-delay: var(--spectrum-animation-duration-200);
+
+	/* TODO update to --spectrum-overlay-color token once it is pointing to correct black RGB token */
+	--spectrum-underlay-background-color: rgba(var(--spectrum-black-rgb), var(--spectrum-overlay-opacity));
+
+	/* --spectrum-underlay-background-color: rgba(var(--spectrum-overlay-color), var(--spectrum-overlay-opacity)); */
```

Comment:
The design specs say to use `overlay-color` and `overlay-opacity`. `overlay-color`, however, isn't pointing to the correct RGB token for black. I decided to leave comments about it all instead, but I can remove/reword whatever if anybody thinks I shouldn't leave this in.